### PR TITLE
feat(pci.rancher): add rancher id

### PR DESCRIPTION
**Pull request by ppprevost**
ref: TAPC-1024

| Question         | Answer
| ---------------- | ---
| Branch?          | `master` / `release/**` / `develop` 
| Bug fix?         | yes/no
| New feature?     | yes/no
| Breaking change? | yes/no
| Tickets          | Fix #... 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by fredericvilcot — DISMISSED** [review]
👍🏻

File: packages/manager/apps/pci-rancher/src/components/Table/DataGridCell.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Clipboard, DataGridTextCell } from '@ovhcloud/manager-components';
+import { Cell } from '@tanstack/react-table';
+import { RancherService } from '../../api/api.type';
+import './Table.scss';
+
+interface DisplayCellInterface {
+  cell: Cell<RancherService, unknown>;
```
Comment (y4nnL) [inline packages/manager/apps/pci-rancher/src/components/Table/DataGridCell.tsx:8]:
```suggestion
  readonly cell: Cell<RancherService, unknown>;
```

**Review by Eric-ciccotti — DISMISSED** [review]
do not forget to complet the MR :)

File: packages/manager/apps/pci-rancher/src/components/Table/DataGridCell.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+const ClipboardInput = ({ cell }: Readonly<DisplayCellInterface>) => {
```
[the remark this answers is not in the document]
Comment (fredericvilcot) [reply packages/manager/apps/pci-rancher/src/components/Table/DataGridCell.tsx]: @y4nnL For my own understanding, what's the difference between both? 
The `<Readonly>` type comes to mark all properties as `readonly`, as equivalent to `readonly cell`.
Thanks!

**Review by y4nnL — APPROVED** [review]

File: packages/manager/apps/pci-rancher/src/components/Table/DataGridCell.tsx
Comment (anooparveti) [inline packages/manager/apps/pci-rancher/src/components/Table/DataGridCell.tsx:1]: **nitpick:** Use appropriate names for Components. Name suggests that the component is for DatagridCell. But it is very specific to display clipboard in the datagrid.

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by Tsiorifamonjena — COMMENTED** [review]
@ppprevost  the target branch was rebased, and there are conflicts. Can you resolve them please ?

**Pull request closed without merging**